react/src/main.js

```diff
@@ -1,10 +1,11 @@
 import 'babel-polyfill';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import ParkShow from './containers/ParkShow';
 
 $(function() {
   ReactDOM.render(
-    <App />,
+    <ParkShow />,
     document.getElementById('app')
   );
```

Comment:
Um, this suggest that your React app will always render a show component first, but it also is a non-working one without a working fetch request on componentDidMoment. This does not make sense. Just render an empty Index component to start